src/content/blog/val-vibes.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Val Vibes: Semantic search in Val Town"
+description: How to build semantic search for Val Town within Val Town itself
+pubDate: June 18, 2024
+author: JP Posma
+---
+
+import { Image } from "astro:assets";
+import ManBitesDog from "./val-vibes/man-bites-dog.png";
+import CompareEmbeddings from "./val-vibes/compare-embeddings.png";
+import QueryVals from "./val-vibes/query-vals.png";
+import Indexing1 from "./val-vibes/indexing1.png";
+import Indexing2 from "./val-vibes/indexing2.png";
+import Indexing3 from "./val-vibes/indexing3.png";
+import Search1 from "./val-vibes/search1.png";
+import Search2 from "./val-vibes/search2.png";
+import Blobindex from "./val-vibes/blobindex.png";
+import Blobsearch from "./val-vibes/blobsearch.png";
+import ManLicksDog from "./val-vibes/man-licks-dog.jpg";
+
+### Similar vibes
+
+“Search” is finding documents based on a query, using some sort of sorting mechanism that ranks the most relevant documents. One such sorting mechanism is “semantic search”, which uses one of those crazy features to come out of the world of machine learning: embeddings. It works like this: you put in a string into an API, and you get an array of numbers (the “embedding”), with the property that things that have the same “vibe” will get similar numbers.
```

Comment:
I think it could make sense to have another sentence or two on what embeddings are